app/views/users/show.html.erb

```diff
@@ -1,60 +1,61 @@
 <% flash.each do |type, msg| %>
-  <div class="alert alert-info">
-    <%= msg %>
-  </div>
+<div class="alert alert-info">
+  <%= msg %>
+</div>
 <% end %>
 
 <div class="container">
-  <div class="row">
-    <div class="mt-5">
-      <h1 class="mb-0"><%= @user.first_name %> <%= @user.last_name %></h1>
-      <p class="mb-0">Email: <%= @user.email %></p>
-      <p>Major: <%= @user.major %></p>
+  <div class="mt-5">
+    <h1 class="mb-0"><%= @user.first_name %> <%= @user.last_name %></h1>
+    <p class="mb-0">Email: <%= @user.email %></p>
+    <p class="mb-0">Major: <%= @user.major %></p>
+    <% if current_user.tutor? %>
```

Comment:
this should be @user, not @current_user. Current_user may be an admin viewing the user's profile

Reply:
Fixed in latest push